launcher/FileSystem.cpp

```diff
@@ -770,9 +770,37 @@ QString getDesktopDir()
 bool createShortcut(QString destination, QString target, QStringList args, QString name, QString icon)
 {
 #if defined(Q_OS_MACOS)
-    destination += ".command";
+    // Create the Application
+    QDir applicationDirectory = QStandardPaths::writableLocation(QStandardPaths::ApplicationsLocation) + "/Prism Launchers/";
```

Comment:
```suggestion
    // Create the Application
    QDir applicationDirectory = QStandardPaths::writableLocation(QStandardPaths::ApplicationsLocation) + "/Prism Launcher Instances/";
```

or something maybe?

Reply:
Maybe also prism launcher should be rather LAUNCHER_DISPLAYNAME